packages/ckeditor5-html-support/CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
 Contributing
-============
+========================================
```

Comment:
Since headers in packages other than emoji are updated anyway in this PR, so maybe let's do it properly once and for all?